.github/workflows/teardown-oss-feature-branch.yml

```diff
@@ -0,0 +1,45 @@
+name: Remove Docker Images and Uninstall Helm Release
+on:
+  workflow_dispatch:
+    inputs:
+      version-postfix:
+        description: "Postfix for the default version: 0.0.0-rc. As an example: 0.0.0-rc99"
+        required: true
+        type: string
+
+env:
+  PRERELEASE_TAG: 0.0.0-rc
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_KEY }}
+  AWS_DEFAULT_REGION: "us-east-1"
+
+jobs:
+  remove-namespace:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+  
+      - name: Delete Kubernetes Namespace
+        uses: kodermax/kubectl-aws-eks@main
+        with:
+            args: delete ns rc${{ inputs.version-postfix }}
+        env:
+          KUBE_CONFIG_DATA: ${{ secrets.KUBECONFIG_BASE }}
+
+  uninstall-helm:
```

Comment:
we should be able to delete the `uninstall-helm` job. 

once all the resources in a namespace are deleted, I think _helm_  removes it from its tracked list of applications too.